h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeNode.java

```diff
@@ -609,4 +610,17 @@ public final int getRightChildIndex() {
     return rightChild != null ? rightChild.internalId : -1;
   }
 
+    public float getGain() {
+        if (!Float.isNaN(gain)) {
```

Comment:
is this because the same code handles XGB and GBM?

if so it would be better to have a flag, eg.: useSEforGain and switch on that

Reply:
yes, flag will be better